includes/admin/payments/class-payments-table.php

```diff
@@ -247,6 +247,18 @@ public function get_sortable_columns() {
 	}
 
 	/**
+	 * Gets the name of the default primary column.
+	 *
+	 * @since 2.5
+	 * @access protected
+	 *
+	 * @return string Name of the default primary column.
+	 */
+	protected function get_default_primary_column_name() {
+		return 'ID';
```

Comment:
I think I disagree here. Payments are made in chronological order, and setting the email as the primary column doesn't achieve us much, as it's not something you'd really sort on either. The ID is the unique value in the payments world, not the email.